oauth2/oauth2-client/src/main/java/org/springframework/security/oauth2/client/web/method/annotation/OAuth2AuthorizedClientArgumentResolver.java

```diff
@@ -105,6 +105,11 @@ public Object resolveArgument(MethodParameter parameter,
 					"@RegisteredOAuth2AuthorizedClient(registrationId = \"client1\").");
 		}
 
+		ClientRegistration clientRegistration = this.clientRegistrationRepository.findByRegistrationId(clientRegistrationId);
```

Comment:
Based on my explanation in this [comment](https://github.com/spring-projects/spring-security/pull/6839#discussion_r281656882), we don't need any changes here as the current implementation does exactly what is required. Please reset the changes here.